### DISCUSSION TITLE
How to Add and Config Custom Certificates as Default

### DISCUSSION THREAD

**Discussion by heirro** [Dokploy/dokploy#551] [Q&A]
Hi,

I just recently installed Dokploy on my Server, and planned installed multi-server in One Panel, but didn't found setup Own Certificates for my Domains. Dokploy docs just Traefik config, but on the Dokploy too complicated for implementation.

How to show custom certificates by owned, when deploying Projects/Services as default?

![Screenshot_20241013](https://github.com/user-attachments/assets/9a18bed3-0b81-4990-87fb-3a8f1abbb680)

![Screenshot_20241013_1](https://github.com/user-attachments/assets/638a5759-3534-48f2-9067-e0baec614545)

**Comment by Siumauricio** [comment]
Read this: https://github.com/Dokploy/dokploy/issues/145#issuecomment-2227908514

**Reply by heirro** [reply]
Hmm, that's not that helpful, what if i want to install my own Wildcard SSL on the Admin Panel 
 (e.g panel.domain.com) and Projects/Service using the same primary sub-domain like .eg project1.domain.com

And how is the `/dashboard/traefik` configured? Is it necessary to make manual edits to `/etc/dokploy/traefik/traefik.yml` or `/etc/dokploy/traefik/dynamic/dokploy.yml` after that?

![Screenshot_20241014](https://github.com/user-attachments/assets/4a207da3-d2d5-44fd-8c96-b5a372417200)
![Screenshot_20241014_1](https://github.com/user-attachments/assets/f6fcf0b9-9fb2-4afd-a00c-7b9a16b6cc43)

**Comment by TechnoDD** [comment]
Hi @heirro 

The problem is that the content of the host folder /etc/dokploy/certificates is not mapped inside the Traefik container. By manually copying the content of the /etc/dokploy/certificates folder into the Traefik container at /etc/dokploy/traefik/dynamic/certificates and adding the certificates to the dokploy.yml configuration, it works.

**Reply by jrparks** [reply]
Would you show the config, maybe a few screenshots?

**Comment by nhridoy** [comment]
Does anyone have any solution?
I have added a certificate in the certificate menu. However, when trying to set that certificate for the Dokploy domain, there is only Let's Encrypt, similar to the project's domain option.

<img width="1326" height="572" alt="Image" src="https://github.com/user-attachments/assets/0b0bedfe-74bd-4d7a-b28e-aa1cd7064fa3" />

Server domain

<img width="1324" height="559" alt="Image" src="https://github.com/user-attachments/assets/799e7306-f971-442c-9ccf-d90fe045acc4" />

<img width="1343" height="552" alt="Image" src="https://github.com/user-attachments/assets/69e9d458-e7ad-4f09-8646-f1cebf8f8787" />

Project domain

I do have a certificate in the certificate menu:

<img width="1348" height="539" alt="Image" src="https://github.com/user-attachments/assets/87a1c0ae-c8c0-402f-a10e-70534205dc60" />
